local_server/qwen3_tts_server/local_server.py

```
@@ -0,0 +1,261 @@
+                torch_dtype=dtype,
+                attn_implementation="flash_attention_2" if self.device == "cuda" else "eager",
+                low_cpu_mem_usage=True
+            ).to(self.device)
+
+            # C. 封装推理包装器
+            self.model = Qwen3TTSModel(model=raw_model, processor=processor)
+            self.pad_token_id = raw_model.config.pad_token_id or raw_model.config.eos_token_id
+
+            # D. 预加载/导出音色特征 (秒开优化)
+            if os.path.exists(self.pt_path):
+                logger.info(f"✨ 发现音色特征 {self.pt_path}，执行秒速加载")
```

Ruff (0.14.14)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)